components/core/Accordian.js

```diff
@@ -18,8 +18,8 @@ class Accordian extends AbstractComponent {
         this.props["expandedItems"] = items;
     }
 
-    setSingleExpansion(boolean) {
-        this.props["singleExpansion"] = boolean;
+    setSingleExpansion(expansion) {
```

Comment:
The property on the component itself is a little confusing... I'm assuming this means that only one accordion item can be open at a time. In this case, you want the property name to instruct the caller on what value to provide (especially as JS lacks strict-typing). Let's adjust to `single` as the argument name, to imply a boolean decision.